### DISCUSSION TITLE
Flash of badger db outbut in my tui

### DISCUSSION THREAD

**Discussion by andrewebdev** [charmbracelet/bubbletea#433] [Q&A]
When using badger db with my TUI, whenever I'm performing read/write operations to the badger DB, the TUI will show a quick flash of the output from badger. It's extremely quick and soon as everything is ready the output disappears and the TUI view renders as normal. 

While not a deal-breaker, it's not a nice user experience. Is there a way to run off this behaviour and not see that "logging" output it in the TUI at all?

**Comment by bashbunni** [comment]
Hey @andrewebdev do you have anything we could take a look at to try and troubleshoot?
